howso/direct/client.py

```diff
@@ -2886,6 +2896,15 @@ def react(  # noqa: C901
                 " following values - ['min', 'max', 'most_similar',]"
             )
 
+        if details is not None and 'robust_computation' in details:
+            details['robust_influences'] = details['robust_computation']
+            details['robust_residuals'] = details['robust_computation']
```

Comment:
See my note above in the docstrings. I don't believe that we should auto-set `robust_influences` to match use of the old parameter. That functionality didn't even exist when they initially used `robust_computation`.

Reply:
I understand better that your approach is great.

Reply:
robust_computation=True is the equivalent of setting both the new flags to True